qa/L0_backend_identity/identity_test.py

```diff
@@ -173,18 +173,23 @@
             ("identity_uint32", np.uint32, [8, 5]),
             ("identity_nobatch_int8", np.int8, [0]),
             ("identity_nobatch_int8", np.int8, [7]),
-            ("identity_bytes", object, [1, 1])):
+            ("identity_bytes", object, [1, 1]),
+            ("identity_bf16", "BF16", [1, 1])):
             # yapf: enable
             if np_dtype != object:
                 input_data = (16384 * np.random.randn(*shape)).astype(np_dtype)
+            # FIXME: Add proper bfloat16 support instead of fp16 WAR here
+            elif np_dtype == "BF16":
+                input_data = (16384 * np.random.randn(*shape)).astype(np.float16)
```

Comment:
@jbkyang-nvi this conversion for triton<->np type is a temporary WAR to test identity backend with existing L0_backend_identity. Once proper python client support for bf16 is added, you should tweak this.

Reply:
This WAR probably won't work without modifying the client library to specially handle BF16?

Reply:
I'll see if the WAR can solely be done on test side without dummy implementation in client